share/poly.go

```diff
@@ -14,10 +14,16 @@ import (
 	"crypto/subtle"
 	"encoding/binary"
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 )
 
+type Suite interface {
```

Comment:
Indeed, when requiring only one additional member, it's relatively easy to code it.